Lib/fontTools/varLib/merger.py

```diff
@@ -87,6 +87,11 @@ def mergeLists(self, out, lst):
 
 	def mergeThings(self, out, lst):
 		try:
+			if hasattr(out, "ensureDecompiled"):
```

Comment:
they should always have `ensureDecompiled` as they are all subclasses of otTables.BaseTable, no?

Reply:
No, `mergeThings()` also gets called with the toplevel GDEF/GPOS/GSUB table objects, which do not have it. It's probably true once we're at the `font["GDEF"].table` level, though.

Reply:
oh, I see. We might as well add `ensureDecompiled` to BaseTTXConverter.

Reply:
I just tried your suggestion, but it doesn't work, as `mergeThings()` also gets called with ints, lists, None, and maybe more.

Reply:
ok.. that's why the method name is generic "things". hasattr is fine then